modules/ROOT/pages/manage-backup-restore.adoc

```diff
@@ -1,63 +1,93 @@
 = Back up and Restore Runtime Fabric
 
-To preserve applications in the event of system failure, ensure that Runtime Fabric is automatically backed up. 
-MuleSoft highly recommends configuring hourly backups. Store your backup archive on external storage outside 
-the Runtime Fabric cluster.
+To preserve applications in the event of system failure, confirm that Runtime Fabric is automatically backed up. When scheduling backups, follow best practices by configuring hourly backups. Store your backup archive on external storage outside the Runtime Fabric cluster.
+
+The state of Runtime Fabric is distributed across the Kubernetes cluster. Use the following procedures to back up and restore the required state necessary for all deployed applications, management services, and their configurations. Individual node backup and restore operations are not supported. 
+
+[WARNING]
+====
+Configuration changes to deployed applications and management services made after a back up cannot not be restored.  
+====
 
 [NOTE]
-The state of Runtime Fabric is distributed across its controller nodes. Use the following procedures to back up and restore
-the required state necessary for all deployed applications, management services, and their configurations. 
-Individual node backup and restore operations are not supported.
+====
+For Runtime Fabric running on VMs / Bare Metal, the backup process also includes backing up of underneath Kubernetes appliance related custom resources [logforwarder, smtp, alert and alerttarget].  
```

Comment:
For clarity, maybe "...also includes backing up the custom resources underneath the Kubernetes appliance"? 
I can't quite figure out what's underneath what and what's related to what...

Reply:
Runtime Fabric runs on top of Kubernetes Appliance (called Gravity)  and we backing up custom resources done on the Kubernetes appliance. I am not sure if we can mention our Vendor Gravity in the documentation.

Reply:
No, we are not supposed to mention Gravity in the PCE/RTF docs. The only place it is exposed in the docs is in the name of one of the command line utilities (gravity).